src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -43,18 +43,32 @@ export const ConfirmationModal = ({
   onConfirm,
   ...props
 }: Props): ReactElement => {
+  const handleClick =
+    (action: Function | null | undefined) =>
```

Comment:
I think this could be slightly improved by using a generic so the type of the function can be inferred.
```suggestion
  const handleClick =
    <A extends Function>(action: A) =>
```